.travis.yml

```diff
@@ -17,6 +16,9 @@ before_install:
   - "export DISPLAY=:99.0"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
+install:
```

Comment:
node 4.x and 5.x, on travis at least, seem to come already with a compatible version of node.js. ( https://travis-ci.org/mozilla-jetpack/jpm/jobs/96813602#L100 )

Upgrading npm is a good idea on some systems but I don't think it's necessary on travis - i'd prefer to use travis's recommended npm version for that version of node.